src/qt/rpcconsole.cpp

```diff
@@ -806,20 +806,25 @@ void RPCConsole::clear(bool keep_prompt)
             ).arg(fixedFontInfo.family(), QString("%1pt").arg(consoleFontSize))
         );
 
-#ifdef Q_OS_MAC
-    QString clsKey = "(⌘)-L";
-#else
-    QString clsKey = "Ctrl-L";
-#endif
-
-    message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(PACKAGE_NAME) + "<br>" +
-                        tr("Use up and down arrows to navigate history, and %1 to clear screen.").arg("<b>"+clsKey+"</b>") + "<br>" +
-                        tr("Type %1 for an overview of available commands.").arg("<b>help</b>") + "<br>" +
-                        tr("For more information on using this console type %1.").arg("<b>help-console</b>") +
-                        "<br><span class=\"secwarning\"><br>" +
-                        tr("WARNING: Scammers have been active, telling users to type commands here, stealing their wallet contents. Do not use this console without fully understanding the ramifications of a command.") +
-                        "</span>"),
-                        true);
+    static const QString welcome_message =
+        /*: RPC console welcome message.
+            Placeholders %5 and %6 are style tags for the warning content. */
```

Comment:
Maybe we should expand upon this. args %1,%2,%3,%4 are written space-separated. This sets the precedent that these arguments are either followed or preceded with space. We should make it explicitly clear that %5 and %6 are not space separated from the rest of the text in this translator comment.

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui/pull/331#issuecomment-841674393).